appinfo/info.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <info>
 	<id>twofactor_nextcloud_notification</id>
-	<name>TwoFactor via nextcloud notification</name>
+	<name>Two Factor Authentication via Nextcloud notification</name>
```

Comment:
```suggestion
	<name>Two-Factor Authentication via Nextcloud notification</name>
```

Reply:
Heh good catch! Actually thought about that the other day but forgot again. ;)

@nickvergessen now needs your review. :)